### DISCUSSION TITLE
No response to ticket

### DISCUSSION THREAD

**Discussion by Joseffp** [community/community#36918] [Other Feature Feedback, Questions, & Ideas]
I have been waiting for 8 days for a response to my ticket. I opened one #1830686 and then opened another hoping this would make things happen faster (#1832065). I've spoken with someone on the sales side multiple times and he's said he has flagged my tickets/flagged this internally but still no response.

I am writing this here in hopes it will help. Frankly it doesn't feel right to have to wait this long for a response. I've seen others have similar issues with even longer wait times. I feel I'm in a tough position as my boss is waiting for this information and I'm doing my best to get it but I keep having to explain that no one is responding. I feel I and others should receive responses in a more timely manner. Many thanks.

**Comment by jpbassalot** [comment]
I'm in a similar situation, 8 days as well (#1848784). Upvoting to see if this can get some visibility.
